models/user_repository.go

```diff
@@ -0,0 +1,37 @@
+package models
```

Comment:
The app.* based Identity repository is wrong and should be replaced by the account one. 

But, you've added a 3. UserRepo?!